dev-server/documents/html/vitalsource-pdf.mustache

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>VitalSource book reader (PDF)</title>
+    <style>
+      body {
+        font-family: sans-serif;
+      }
+    </style>
+    <script type="module">
+      /**
+       * Custom element that holds the book content frame. The class name is not
+       * important but the `<mosaic-book>` tag name triggers the VitalSource integration.
+       */
+      class MosaicElement extends HTMLElement {
+        constructor() {
+          super();
+          this.attachShadow({ mode: 'open' });
+
+          // For now there is only one page.
+          const pageURLs = [
+            '/document/vitalsource-pdf-page'
+          ];
+
+          let pageIndex = 0;
+
+          const setPage = index => {
+            if (index < 0 || index >= pageURLs.length) {
+              return;
+            }
+            pageIndex = index;
+
+            // We remove the current frame and create a new one, rather than just
+            // change the `src` of the existing iframe, to mimic what Bookshelf
+            // does. The client should be robust to either approach.
+            this.contentFrame?.remove();
+            this.contentFrame = document.createElement('iframe');
+            this.contentFrame.src = pageURLs[pageIndex];
+            this.shadowRoot.append(this.contentFrame);
+          };
+
+          const styles = document.createElement('style');
```

Comment:
```suggestion
          const style = document.createElement('style');
```

Reply:
Hmm ... this variable refers to one `<style>` element that contains multiple CSS style rules. I was thinking of the latter when I named it.